### PR TITLE
Correct queue behaviour

### PR DESCRIPTION
**Pull request by thorduragust**
The queue reacts to skip commands as expected now. This is a little hacky for now as the stop() function triggered the "after" function of the playing item, so skip would skip twice. To fix this the current stream is just paused, and another stream takes over as the playing one, but the old one still takes up memory I think. I have an idea to fix this but this will do for now. Due to the "after" behaviour described above, the stop command is effectively an alias for skip as is 🤕 
There is also a queue command now that shows you what is in the queue.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -136,25 +137,42 @@ async def on_message(self, message):
 
     def next_in_queue(self, _):
         """Switch to next song in queue"""
-        if not self.song_queue.empty():
-            self.voice_client.stop()
-            self.voice_client.play(self.song_queue.get(), after=self.next_in_queue)
+        if self.play_ctx_queue.empty() == False:
```
Comment (MikkMakk88) [inline bot.py:140]: Is there a reason to change this line from 
```if not self.song_queue.empty():```
to 
```if self.play_ctx_queue.empty() == False:``` ?

File: bot.py [hunk 2]
```diff
@@ -195,6 +213,22 @@ async def skip(self, _message, _command_content):
...
 
+    async def queue(self, message, command_content):
+        """Displays media that has been queued"""
+        reply = ""
+        items = list(self.play_ctx_queue.queue)
+
+        if len(items) == 0:
+            reply = "No audio in queue."
+
+        for i in range(0, len(items)):
```
Comment (MikkMakk88) [inline bot.py:224]:
```suggestion
        for i in range(len(items)):
```

**Comment by morgaesis** [comment]
Very good stuff :)

Take a pass with the formatter and linter when you are done. Otherwise, banger PR.

File: bot.py [hunk 3]
```diff
@@ -195,6 +213,22 @@ async def skip(self, _message, _command_content):
         """Skip to next song in queue"""
         self.next_in_queue(None)
 
+    async def queue(self, message, command_content):
+        """Displays media that has been queued"""
+        reply = ""
+        items = list(self.play_ctx_queue.queue)
+
+        if len(items) == 0:
```
[the remark this answers is not in the document]
Comment (thorduragust) [reply bot.py:221]: I also agree that `if not items` is less explicit, but I don't particularly care.

[the remark this answers is not in the document]
Comment (morgaesis) [reply bot.py:221]:
```python
if not "non-empty string":
    # True
if not "":
    # False
```

File: bot.py (hunk 2 above)
Comment (morgaesis) [reply bot.py:224]: Linter will complain about `range(len(...))` anyway.

**Pull request merged**